# xfm status msg
Header header
string xfm_fm_version # xfm firmware version
string xfm_build_num # xfm firmware build number
int32 xfm_wakeup_angle # xfm wake-up angle
int32 xfm_wakeup_score # xfm wake-up score